shortfin/python/shortfin/array/__init__.py

```diff
@@ -83,3 +85,10 @@
     "fill_randn",
     "RandomGenerator",
 ]
+
+# Import nputils if numpy is present.
+np_present = importlib.util.find_spec("numpy") is not None
+if np_present:
+    from . import _nputils as nputils
+
+    __all__.append("nputils")
```

Comment:
I have a feeling that this will not play well with static type analyzers, but I guess we will find out.